source/features/pinned-issues-update-time.tsx

```diff
@@ -25,7 +25,9 @@ const getLastUpdated = cache.function(async (issueNumbers: number[]): Promise<Re
 
 	return repository;
 }, {
-	maxAge: 1,
+	maxAge: {
+		days: 1
```

Comment:
```suggestion
		minutes: 30
```

This almost should not be cached